tools/docker/windows/install_intellij.sh

```diff
@@ -0,0 +1,46 @@
+#!bin/bash
+
+ed=C
+
+# Fetch the most recent version
+VERSION=$(wget "https://www.jetbrains.com/intellij-repository/releases" -qO- | grep -P -o -m 1 "(?<=https://www.jetbrains.com/intellij-repository/releases/com/jetbrains/intellij/idea/BUILD/)[^/]+(?=/)")
+
+# Prepend base URL for download
+URL="https://download.jetbrains.com/idea/ideaI$ed-$VERSION.tar.gz"
+
+echo $URL
+
+# Truncate filename
+FILE=$(basename ${URL})
+
+# Set download directory
+DEST=~/Downloads/$FILE
+
+echo "Downloading idea-I$ed-$VERSION to $DEST..."
+mkdir ~/Downloads/
+
+# Download binary
+wget -cO ${DEST} ${URL} --read-timeout=5 --tries=0
+
+echo "Download complete!"
+
+# Set directory name
+DIR="/opt/idea-I$ed-$VERSION"
+
+echo "Installing to $DIR"
+
+# Untar file
+if mkdir ${DIR}; then
```

Comment:
* Either drop this `if`, or make all such commands robustly exit with an error if they fail.
* Quotes.
* No `{}`s.